app/furniture/marketplace/marketplace_component.html.erb

```diff
@@ -1,18 +1,5 @@
-<h2 class="ml-2 my-2">
-  <%= t('.header', space: marketplace.space.name, room: marketplace.room.name) %>
-</h2>
+<div class="grid grid-cols-1 gap-6">
+  <%= render delivery_area_component %>
 
-
-<%= render onboarding_component %>
```

Comment:
So now this will only show in the `ManagementComponent` right? Nice.

Reply:
Yep!